molo/profiles/models.py

```diff
@@ -325,6 +326,18 @@ class UserProfile(models.Model):
         through="SecurityAnswer"
     )
     site = models.ForeignKey(Site, blank=True, null=True)
+    migrated_username = models.CharField(
+        _('migrated_username'),
+        max_length=30,
```

Comment:
Is there a particular reason we're using 30 chars here?

Reply:
to match the original username, but going to make it 40